server/services/interventionsService.ts

```diff
@@ -26,15 +26,16 @@ export interface ReferralFields {
   interpreterLanguage: string | null
   hasAdditionalResponsibilities: boolean
   whenUnavailable: string | null
-  serviceUser: ServiceUser
   additionalRiskInformation: string
   usingRarDays: boolean
   maximumRarDays: number | null
+  serviceUser: ServiceUser
```

Comment:
Can we undo this move?

Reply:
Yep, moved this by mistake, it's back now.